src/GraphQL/GraphQL.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL-Parser" Version="5.3.0" />
+    <PackageReference Include="GraphQL-Parser" Version="5.3.3" />
```

Comment:
The main motivation was updating the parser, which began to allocate much less memory.

Reply:
With a "side effect" of fixing a bug where syntax error messages show inaccurate locations

Reply:
Of course